assets/prototype/application/ui/components/display/entityListFilterBar/useEntityListFilterState.js

```diff
@@ -0,0 +1,102 @@
+import { useReducer, useEffect } from '@wordpress/element';
+import { assocPath, path } from 'ramda';
+
+const useEntityListFilterState = (listId) => {
+	const initialState = {
+		[listId]: {
+			perPage: 6,
+			searchText: '',
+			showEntityFilters: false,
+			view: 'list',
+		},
+	};
```

Comment:
I don't think `listId` is needed here. When you instantiate that hook, datetime and tickets will get their own instance, that's the best part of hooks in such a situation.

Reply:
Done, thx :bowing_man:

Reply:
Need to think away from redux model of single object state.